tsconfig.json

```diff
@@ -16,7 +16,7 @@
     "index.ts",
     "bin/**/*.ts",
     "src/**/*.ts",
-    "test/**/*.ts"
+    "test/**/*.ts",
```

Comment:
@thdossch this is a long shot but could you see if changing this line back fixes it? The extra comma there might cause the tsconfig.json to be invalid, causing the compiler to not see which files it has to use anymore and thereby trying to include all files (and causing that error you have now).